notebooks/official/feature_store/online_feature_serving_and_fetching_bigquery_data_with_feature_store_optimized.ipynb

```diff
@@ -90,20 +90,6 @@
         "- Use the online server to fetch feature values for online prediction."
```

Comment:
<p>please use <code>id</code> for now, because optimized support <code>data_key</code>. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2495/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='79'/>